Minio/ApiEndpoints/BucketOperations.cs

```diff
@@ -368,6 +368,50 @@ public IObservable<MinioNotificationRaw> ListenBucketNotificationsAsync(ListenBu
         }
 
 
+        /// <summary>
+        /// Sets the notification configuration for this bucket
+        /// </summary>
+        /// <param name="args">SetObjectLockConfigurationArgs Arguments Object with information like Bucket name, object lock configuration to set</param>
+        /// <param name="cancellationToken">Optional cancellation token to cancel the operation</param>
+        /// <returns></returns>
+        public async Task SetObjectLockConfigurationAsync(SetObjectLockConfigurationArgs args, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            args.Validate();
+            RestRequest request = await this.CreateRequest(args).ConfigureAwait(false);
+            await this.ExecuteAsync(this.NoErrorHandlers, request, cancellationToken).ConfigureAwait(false);
+        }
+
+
+         /// <summary>
+        /// Gets notification configuration for this bucket
+        /// </summary>
+        /// <param name="args">GetBucketNotificationsArgs Arguments Object with information like Bucket name</param>
+        /// <param name="cancellationToken">Optional cancellation token to cancel the operation</param>
+        /// <returns>ObjectLockConfiguration object</returns>
+        public async Task<ObjectLockConfiguration> GetObjectLockConfigurationAsync(GetObjectLockConfigurationArgs args, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            args.Validate();
+            RestRequest request = await this.CreateRequest(args).ConfigureAwait(false);
+            IRestResponse response = await this.ExecuteAsync(this.NoErrorHandlers, request, cancellationToken).ConfigureAwait(false);
+            GetObjectLockConfigurationResponse getBucketNotificationsResponse = new GetObjectLockConfigurationResponse(response.StatusCode, response.Content);
+            return getBucketNotificationsResponse.LockConfiguration;
+        }
+
+
+        /// <summary>
+        /// Sets the notification configuration for this bucket
```

Comment:
```suggestion
        /// Removes the notification configuration for this bucket
```

Reply:
Sorry for the oversight. Corrected the mistakes. PTAL